assets/styles/app.css

```diff
@@ -22,11 +22,46 @@ body {
 }
 
 .hamburger {
-  display: block;
-  position: relative;
+  background-image: url(/assets/icons/menu.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  width: 45px;
+  height: 30px;
   z-index: 1;
 }
 
+.menu-contaner {
+  gap: 10rem;
+}
+
+.hamburger.active {
+  background-image: url("/assets/icons/x.svg");
+  background-size: contain;
+}
+
+.header-d {
+  position: fixed;
+  left: -100%;
+  width: 100%;
+  height: 100%;
+  padding: 3rem;
+  transition: 0.2s;
+  list-style: none;
+  background: #fff;
+  mix-blend-mode: normal;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+}
```

Comment:
- Hey, let's kindly add a z-index of 2 to ensure the mobile menu stays on tope of other sections in your web page to improve the user experience.